atlas_densities/densities/measurement_to_density.py

```diff
@@ -69,8 +69,11 @@ def compute_region_densities(
         The index is the sorted list of all region identifiers.
     """
     densities = []
-    for set_ in hierarchy_info["descendant_id_set"]:
-        mask = np.isin(annotation, list(set_))
+    tot = len(hierarchy_info["descendant_id_set"])
+    print("\nDensities to compute:", tot)
+    for iset_ in range(tot):
+        if not (iset_ % (tot/10)): print("density %d of %d" % (iset_+1, tot))
```

Comment:
Note this is failing CI; single line `if:  statement` is not allowed.

Please don't use short names if it's not required (ie: `tot` vs `total`)

Can't we use tqdm or click.progress bar?

Reply:
Addressed in 92918c90fc79246179609e2d2a89da31223025ca